cluster/pulumi/common-sv/src/config.ts

```diff
@@ -99,3 +102,13 @@ export type Config = z.infer<typeof SvConfigSchema>;
 // eslint-disable-next-line
 // @ts-ignore
 export const svConfig = SvConfigSchema.parse(clusterYamlConfig).sv;
+
+export const updateHistoryBackfillingValues = svConfig?.scan?.enableImportUpdatesBackfill
+  ? {
+      updateHistoryBackfilling: {
+        enabled: true,
+        importUpdatesEnabled: true,
+        batchSize: 100,
+      },
+    }
+  : {};
```

Comment:
```suggestion
  : undefined;
```

This _should_ be the same in helm but I think undefined better simulates people not setting the value.